exotic.py

```diff
@@ -2246,6 +2252,14 @@ def lc2min(x):
                     # Exit aperture loop
                 # Exit annulus loop
             # Exit the Comp Stars Loop
+            #Doublechecking to see if comparison star != variable star
```

Comment:
I would move this up to the top of the `for` loop- that way, if a comp star is variable, you throw it away immediately- and don't waste time doing processes such as extracting its photometry - and move on right away to the next comp star.

Reply:
That makes sense since we could save time by not having to go through twice to see which stars are variable instead of throwing them out before they ever get the chance. I'll figure out a better place to move the command to in the loop.